qcodes/dataset/database.py

```diff
@@ -43,3 +44,14 @@ def initialise_or_create_database_at(db_file_with_abs_path: str) -> None:
     """
     qcodes.config.core.db_location = db_file_with_abs_path
     initialise_database()
+
+
+def path_of_connection(conn: SomeConnection) -> str:
```

Comment:
What about a different name?
```suggestion
def db_file_of_connection(conn: SomeConnection) -> str:
```

Reply:
I fully agree to both of your suggestions @astafan8 !

Reply:
Although I acknowledge the confusion, I will stick with `path_of_connection`, since a path is indeed what this function returns. If that is too unclear, we could also go full out with `path_to_db_file_of_connection`.

Reply:
`db_path_of_connection` ? :) otherwise, i agree with the "full out explicit" one